.github/workflows/tenet-pr-review.yml

```
@@ -14,10 +14,11 @@ jobs:
     name: TENET Security Review
     runs-on: ubuntu-latest
     if: github.actor != 'github-actions[bot]' && github.actor != 'coderabbitai[bot]'
-    steps:
+        steps:
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)